src/panels/lovelace/editor/hui-dialog-edit-card.ts

```diff
@@ -88,13 +107,40 @@ export class HuiDialogEditCard extends LitElement {
     this._dialog.close();
   }
 
+  private _toggleEditor() {
+    if (!this._elementConfig) {
+      return;
+    }
+    this._showUIEditor = !this._showUIEditor;
+    // This will center the dialog with the updated config
+    fireEvent(this._dialog, "iron-resize");
+  }
+
   private async _loadConfig() {
     this._cardConfig = await getCardConfig(this.hass!, this._cardId!);
+    this._loadElementConfig();
     await this.updateComplete;
     // This will center the dialog with the updated config
     fireEvent(this._dialog, "iron-resize");
   }
 
+  private async _loadElementConfig() {
+    const conf = yaml.safeLoad(this._cardConfig);
+    const elClass = customElements.get(`hui-${conf.type}-card`);
+    const element = document.createElement(`hui-${conf.type}-card`);
```

Comment:
not needed when you make it a static method.

Reply:
ah gotcha. Yea I left that `elClass` definition in there so that you could see I tried that but it wasnt working.